### PR TITLE
NIAD-974: update test resources to include practitioner entry

### PR DESCRIPTION
**Pull request by n-r-b**
## Description

In preparation for the patient mapper implementation (ticket attached), I've updated the edifact and fhir test resources to include current practitioner resource.

This has been done in #54, but i thought it may be useful for other mapping tickets if we pull this out into a separate (smaller) PR.

## Jira Ticket

https://gpitbjss.atlassian.net/browse/NIAD-974

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [x] Acceptance Criteria met
- [x] Commit messages are meaningful
- [x] Manually tested
- [x] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [x] If your pull request depends on any other, please link them in the description
- [x] main branch is passing/green on CI
- [x] Add/update any relevant documentation

### CODE REVIEW

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java [hunk 1]
```diff
@@ -85,15 +85,15 @@
     private String gpOutboundQueueName;
 
     @Getter
-    @Value("classpath:edifact/pathology.dat")
+    @Value("classpath:edifact/pathology.edifact.dat")
```
Comment (n-r-b) [inline src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java:88]: Renamed to match file names specified here: https://github.com/nhsconnect/integration-adaptor-lab-results/blob/main/README.md#examples

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java [hunk 2]
```diff
@@ -58,19 +63,15 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions so
...
             .mapToObj(x -> getGpOutboundQueueMessage())
             .collect(Collectors.toList());
 
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(0));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(1));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(2));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(3));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(4));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(5));
+        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(0), fhirMessage1);
```
Comment (n-r-b) [inline src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java:70]: Now we're testing multiple 'different' messages, i've removed 4 messages until we have more mappers in place.

File: src/intTest/resources/edifact/multi_pathology.edifact.dat [hunk 3]
```diff
@@ -0,0 +1,40 @@
+UNB+UNOC:3+XX11+000000024600002:80+100301:1751+1015++MEDRPT++1'
```
Comment (n-r-b) [inline src/intTest/resources/edifact/multi_pathology.edifact.dat:1]: the interchange start example we have is slightly different.

`UNB+UNOC:3+000000004400001:80+000000024600002:80+100301:1751+1015++MEDRPT++1'`

This includes `:80` after the recipient. I've removed this for now, until we've updated the interchange header parser.

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java [hunk 4]
```diff
@@ -58,19 +63,15 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions so
 
     @SuppressWarnings("checkstyle:magicnumber")
     private void assertGpOutboundQueueMessages(SoftAssertions softly) throws IOException, JMSException, JSONException {
-        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 6)
+        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 2)
             .mapToObj(x -> getGpOutboundQueueMessage())
             .collect(Collectors.toList());
```
Comment (andrewwa-kainos) [inline src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java:68]: Not sure the streaming is valuable anymore
```suggestion
        final List<Message> gpOutboundQueueMessages = List.of(getGpOutboundQueueMessage(), getGpOutboundQueueMessage());
```

**Review by andrewwa-kainos — DISMISSED** [review]
LGTM - just one ignorable suggestion

File: src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java (hunk 4 above)
Comment (n-r-b) [reply src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java]: Good spot - thanks!

**Review by andrewwa-kainos — APPROVED** [review]

**Review by matthewdoreilly — APPROVED** [review]

**Pull request merged**